test/perf-test.js

```diff
@@ -0,0 +1,53 @@
+/* eslint-disable no-console */
```

Comment:
TODO: top level docstring explaining the purpose of the file and how to run the tests.

Reply:
Don't forget!